roles/libvirt_manager/tasks/start_manage_vms.yml

```diff
@@ -54,15 +54,14 @@
   delegate_to: localhost
   vars:
     extracted_ip: "{{ (vm_ip.stdout.split())[3] | ansible.utils.ipaddr('address') }}"
-    proxy_hostname: "{{ ansible_host | default(inventory_hostname) }}"
   ansible.builtin.blockinfile:
     create: true
     path: "{{ lookup('env', 'HOME') }}/.ssh/config"
     marker: "## {mark} {{ vm_ip.nic.host }}"
     mode: "0600"
     block: |-
       Host {{ vm_ip.nic.host }} cifmw-{{ vm_ip.nic.host }} {{ extracted_ip }}
-        ProxyJump {{ ansible_user | default(lookup('env', 'USER')) }}@{{ proxy_hostname }}
+        ProxyJump {{ ansible_user | default(lookup('env', 'USER')) }}@{{ inventory_hostname }}
```

Comment:
now I remember why there's that thing.
This is my inventory file:
```YAML
localhosts:
  hosts:
    localhost:
      ansible_connection: local

hypervisors:
  hosts:
    builder1:
      ansible_user: virtuser
      ansible_host: builder1.rh.lain.internux.ch
    builder2:
      ansible_user: virtuser
      ansible_host: builder2.rh.lain.internux.ch
```
As you can see, I'm using aliases, and leverage the `ansible_host` to allow proper DNS resolution (though I don't really need it in my LAN).

If we go straight to use the `inventory_hostname` we'll face issues when people are doing the same as I'm doing,in order to get an easier/faster way to name the `cifmw_target_host` (like I do) while not having DNS resolution for that alias (in my case, `builder-1` and `builder-2`)...

We may want to get some smarter detection of the name to use, buuut.... I'm not really sure how :/.

In your case, adding `ansible_host` to your inventory may solve your issue (alongside the `localhost` entry, to ensure you're using `local` connection)...

Reply:
Done and working now! Thanks Cédric, I'll look at adding an assertion to check for this in case someone else misses it :)